local-cluster/test/Test/EndToEndSpec.hs

```diff
@@ -75,3 +82,10 @@ spec = describe "End-to-end test using a mocked chain though" $ do
 
                 metrics <- getMetrics n1
                 metrics `shouldSatisfy` ("hydra_head_events  3" `BS.isInfixOf`)
+
+  describe "hydra-node executable" $ do
+    it "display proper semantic version given it is passed --version argument" $ do
+      failAfter 5 $
+        withCreateProcess ((hydraNodeProcess ["--version"]){std_out = CreatePipe}) $ \_ (Just out) _ _ -> do
```

Comment:
There is [readCreateProcess](https://hackage.haskell.org/package/process-1.6.11.0/docs/System-Process.html#v:readCreateProcess) which would make accessing `stdout` a bit simpler (for such simple string cases), i.e. `version <- readCreateProces (hydraNodeProcess ["--version"]) ""`